ha-hoymiles-wifi-addon/run.sh

```diff
@@ -31,15 +31,26 @@ if [[ -z "$MQTT_BROKER_HOST" ]]; then
   exit 1
 fi
 
-if [[ -z "$MQTT_USERNAME" ]]; then
-  echo "The mqtt_username is not configured."
-  exit 1
+# The host names are mandatory information
+echo "inverter_host = \"$INVERTER_HOST\" \n\
```

Comment:
I suggest to use

```toml
# Create the configuration file
cat <<EOF > ./config.toml
inverter_host = "$INVERTER_HOST"

[home_assistent]
host = "$MQTT_BROKER_HOST"
username = "$MQTT_USERNAME"
password = "$MQTT_PASSWORD"
port = $MQTT_PORT
EOF
```
instead. It's easier to read and maintain (also there was some toml syntax issue, not sure what exactly, but the above works fine)

Reply:
Thx, changed